sample_repos.md

```diff
@@ -0,0 +1,992 @@
+# Sample Binder Repositories
+
+
+Below we list several sample Binder repositories that
+demonstrate how to compose build files in order to create
+Binders with varying environments. You can find all of the
+repositories listed on this page at the
+[binder-examples GitHub organization](https://github.com/binder-examples).
+
+
+---------
+## Julia Binder demo
+
+This is a demo of Julia functionality for the Binder project. Simply
+go to the URL below and it will launch an interactive Julia environment:
+
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/binder-examples/demo-julia/master?filepath=demo.ipynb) | [repo link](https://github.com/binder-examples/demo-julia)
+
+
+### Files in this repository
+```
+REQUIRE
+demo.ipynb
+```
+```eval_rst
```

Comment:
For my education: what does this do? Mostly I am confused seeing something about rst in a markdown file :)

Reply:
it just treats it as "regular" rst - the recommonmark thing does .md -> rst -> html, so this just lets you inject rst directly in there.